src/utils/feature-flags.ts

```diff
@@ -18,6 +18,7 @@ const defaultFlagSet: LDFlagSet = {
   'supported-agm-location-chg-entities': [],
   'supported-amalgamation-entities': [],
   'supported-amalgamation-out-entities': [],
+  'supported-ar-correction-entities': [],
```

Comment:
Default value for the FF if LaunchDarkly cannot be reached: no entities are allowed to do AR corrections.

Currently:
- in Dev, the FF is "CP"
- in Test, the FF is "CP"
- in Prod, the FF is ""